### PR TITLE
VideoPlayer: fix/improve audio stalls of realtime streams

### PR DESCRIPTION
**Pull request by FernetMenta**
- clock was set wrong for realtime streams
- improve behaviour in case of stalled audio stream

### CODE REVIEW

File: xbmc/cores/VideoPlayer/VideoPlayer.cpp [hunk 1]
```diff
@@ -1900,6 +1900,13 @@ void CVideoPlayer::HandlePlaySpeed()
           (!m_VideoPlayerVideo->AcceptsData() && m_CurrentVideo.id >= 0))
         SetCaching(CACHESTATE_INIT);
     }
+
+    // if audio stream stalled, wait untit demux queue filled 10%
```
Comment (notspiff) [inline xbmc/cores/VideoPlayer/VideoPlayer.cpp]: until

Comment (FernetMenta) [reply xbmc/cores/VideoPlayer/VideoPlayer.cpp]: thx

**Comment by vpeter4** [comment]
May I ask what is the reason for this changes and what does they improve? Because with this PR my live HLS streams from TV provider are loading much slower. Meaning channel switch time with PVR IPTV Simple client is much slower (same with Tvheadend). And I didn't notice any audio issue with this streams.
Tested with official LibreELEC build, builds from Milhouse to identify this PR and my own build.

**Comment by FernetMenta** [comment]
Intended. We go to init state as soon level is above 10

**Pull request merged**